test/bdd/features/importer.feature

```diff
@@ -24,3 +24,17 @@ Feature: Test basic importer features
     And DC imports "importers/xml_examples/Retail/01_Green_to_pink_shipment.xml" as GS1
     And DC imports "importers/json_examples/WOT_Example_1.json" as WOT
     Then the traversal from batch "urn:epc:id:sgtin:Batch_1" should contain 1 trail and 2 vertices of type EVENT
+
+  @local
```

Comment:
I think the scenario should be labeled as @first. Hope there's enough time for this one more.